### ISSUE TITLE
Does the mipsdk work in an aspnet core 5 app hosted in azure?

### ISSUE THREAD

**Issue by jannikbuschke** [Azure-Samples/MipSDK-File-Dotnet-Quickstart#16]
Question in title...

**Comment by HanishaChowdary** [comment]
Yes, the Microsoft Information Protection SDK (MIP SDK) can work with an ASP.NET Core app hosted in Azure. However, there are some considerations and configurations required to ensure compatibility:

1. Azure App Registration: You need to register your application in Azure Active Directory to enable authentication and permissions for the MIP SDK.
2. SDK Version: Ensure you are using a compatible version of the MIP SDK. The latest versions often include updates and fixes for better integration with ASP.NET Core applications.
3. Authentication Flow: Implement an appropriate authentication flow, to allow the application to authenticate users and interact with the MIP SDK.
4. NuGet Packages: Install the necessary NuGet packages for the MIP SDK in your ASP.NET Core project.

For more detailed guidance, explore this https://learn.microsoft.com/en-us/samples/azure-samples/mipsdk-fileapi-dotnet-onbehalfof/mipsdk-file-dotnet-onbehalfof/?form=MG0AV3 that demonstrates using the MIP SDK with ASP.NET applications
